### PR TITLE
Deprecate Support for MagDB

### PR DESCRIPTION
**Pull request by jrha**

### CODE REVIEW

File: views/view-logical-workers.php [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
 
-// Gathers clusters
-$all_clusters = Array();
-$data = pg_query('select "name", "categoryName" from "vBatchAndCloudVMs"');
-if ($data and pg_num_rows($data)){
-    while ($row = pg_fetch_assoc($data)) {
-        $all_clusters[$row['name']] = $row['categoryName'];
-    }
+function is_worker_node($var) {
+    return ((strpos($var, "wn-") !== false) || (strpos($var, "hadoop-") !== false));
```
Comment (jnc74743) [inline views/view-logical-workers.php:11]: As much as I don't like hard coding strings with conditionals, I have no alternative at this time. However, do we need the "hadoop" tranche defined? I'm in favor of removing this reference and decomming the tranches

File: views/view-logical-workers.php [hunk 2]
```diff
@@ -36,57 +42,79 @@
...
+        if ($source !== "Slurm on SCARF") {
+            $results[''][$model][$personality][$hostname] = $details;
+        };
     };
 };
 
 
 // Returns built json
-echo json_encode($results);
+echo json_encode($results) ;
```
Comment (jnc74743) [inline views/view-logical-workers.php:120]: Could the space be removed

File: views/view-logical-workers.php [hunk 3]
```diff
@@ -36,57 +42,79 @@
...
+        };
+
+        $details = Array(
             'status' => $status,
         );
+
+        if (array_key_exists($hostname, $all_notes)) {
+            $details['note'] = $all_notes[$hostname];
+        };
+        if (nagios($hostname) !== Null) {
```
Comment (jnc74743) [inline views/view-logical-workers.php:108]: Should the references to Nagios be removed?

**Review by jnc74743 — CHANGES_REQUESTED** [review]
Nothing major, a query and a linting issue

**Review by jnc74743 — APPROVED** [review]

File: views/view-logical-workers.php (hunk 3 above)
Comment (jrha) [reply views/view-logical-workers.php:108]: Not in this PR.

File: views/view-logical-workers.php (hunk 1 above)
Comment (jrha) [reply views/view-logical-workers.php:11]: Yes, but not in this PR (this is just moved code).

**Pull request merged**